crates/re_viewer_context/src/space_view/space_view_class.rs

```diff
@@ -74,60 +73,15 @@ pub trait SpaceViewClass {
         ctx: &mut ViewerContext<'_>,
         ui: &mut egui::Ui,
         state: &mut dyn SpaceViewState,
-        scene: Scene,
+        scene: Box<dyn Scene>,
     );
 }
 
 /// State of a space view.
 pub trait SpaceViewState: std::any::Any {
```

Comment:
Please document for what `SpaceViewState` is intended for.

It seems this is only used right now as a patch while we are transitioning to blueprints? Then we should mention this here ("prefer to put new state in the blueprint instead…")